packages/plugins/plugin-table/src/components/Table/RowActionsMenu.tsx

```diff
@@ -0,0 +1,38 @@
+//
+// Copyright 2024 DXOS.org
+//
+
+import * as ModalPrimitive from '@radix-ui/react-popper';
+import React from 'react';
+
+import { DropdownMenu, useThemeContext, useTranslation } from '@dxos/react-ui';
+
+import { TABLE_PLUGIN } from '../../meta';
+import { type TableModel } from '../../model';
+
+export type RowActionsMenuProps = {
+  tableModel?: TableModel;
+  rowIndex: number | null;
+  open: boolean;
```

Comment:
optional (esp. since downstream components support optional props). in fact, consider:

```tsx
export type RowActionsMenuProps = {} & Pick<DropdownMenuProps, 'open' | 'onOpenChange'>

export const RowActionsMenu = ({ model, ...props }) => {
  return <DropdownMenu.Root {...props}>
```

Reply:
Will update the props to just forward the underlying `DropdownMenuRootProps` though.